src/libsync/propagateuploadng.cpp

```diff
@@ -360,10 +360,12 @@ void PropagateUploadFileNG::startNextChunk()
         return;
     }
 
-    auto headers = PropagateUploadFileCommon::headers();
+    QMap<QByteArray, QByteArray> headers;
```

Comment:
why this change ?
you are going to miss the headers from `PropagateUploadFileCommon::headers()`

Reply:
From what I have understood from https://github.com/nextcloud/server/blob/26ad0a1b9c4b617e50cfd27b9696b052522953be/apps/dav/lib/Upload/ChunkingV2Plugin.php#L285 and https://github.com/nextcloud/documentation/pull/10740 the key header we need for a chunk upload is `Destination`, at least from the `curl` examples, with `X-OC-Mtime` for the final chunk assembly. It seemed like the PropagateUploadFileCommon::headers() are unnecessary (I added them in the prior fix but they didn't make a difference)

Reply:
ok